src/pages/instances/forms/EditInstanceDetails.tsx

```diff
@@ -76,8 +78,10 @@ const EditInstanceDetails: FC<Props> = ({ formik, project }) => {
       <ProfileSelect
```

Comment:
Good idea about the naming.
As for the read only, we still need that during instance creation, because before the user selects and image, the read only is set to true there.